app/src/main/java/org/fossasia/openevent/app/qrscan/ScanQRActivity.java

```diff
@@ -78,17 +83,19 @@
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
-        OrgaApplication
-            .getAppComponent(this)
-            .inject(this);
-
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_scan_qr);
 
         setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
 
         ButterKnife.bind(this);
 
+        DaggerAppComponent.builder()
+            .androidModule(new AndroidModule(this))
+            .barcodeModule(new BarcodeModule(graphicOverlay))
+            .build()
+            .inject(this);
```

Comment:
I'd recommend that you create a new component for this, so it does not require null in constructor while initializing the app